src/main/java/uk/gov/pay/api/model/PaymentEvent.java

```diff
@@ -13,10 +13,12 @@
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 
-@ApiModel(value="Payment Events information", description = "A List of Payment Events information")
+@ApiModel(value="Payment Event information", description = "A List of Payment Events information")
 public class PaymentEvent {
     @JsonProperty("payment_id")
     private final String paymentId;
+
+    @JsonProperty("status")
```

Comment:
It would pick it up without it. I preferred being explicit on the json names. It doesn't add any overhead, but added for consistency.